src/main/java/pro/sky/telegrambot/model/notification_task.java

```diff
@@ -0,0 +1,53 @@
+package pro.sky.telegrambot.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import java.time.LocalDateTime;
+
+@Entity
+public class notification_task {
```

Comment:
так классы в Java не принято именовать, camelCase использовать должен ты
```suggestion
public class NotificationTask {
```